lib/widgets/nt_widgets/multi-topic/yagsl_swerve_drive.dart

```diff
@@ -93,6 +111,21 @@ class YAGSLSwerveDriveModel extends NTWidgetModel {
           ),
         ],
       ),
+      const SizedBox(height: 5),
+      Row(
+        children: [
+          Flexible(
+            child: DialogTextInput(
+              initialText: _angleOffset.toString(),
+              label: 'Angle Offset ($angleUnit)',
+              onSubmit: (String value) async {
+                angleOffset = double.parse(value);
```

Comment:
I would use a tryParse with a null check in case if the method is somehow called with invalid data. See how it's done for some of the graph settings